src/bake/DevServer/PackedMap.zig

```diff
@@ -31,24 +32,24 @@ end_state: struct {
 /// already counted for.
 bits_used_for_memory_cost_dedupe: u32 = 0,
 
-pub fn newNonEmpty(source_map: SourceMap.Chunk, quoted_contents: []u8) bun.ptr.RefPtr(PackedMap) {
-    assert(source_map.buffer.list.items.len > 0);
+pub fn newNonEmpty(chunk: SourceMap.Chunk, quoted_contents: []u8) bun.ptr.RefPtr(PackedMap) {
+    assert(chunk.buffer.list.items.len > 0);
     return .new(.{
         .ref_count = .init(),
-        .vlq_ptr = source_map.buffer.list.items.ptr,
-        .vlq_len = @intCast(source_map.buffer.list.items.len),
+        .vlq_ptr = chunk.buffer.list.items.ptr,
+        .vlq_len = @intCast(chunk.buffer.list.items.len),
```

Comment:
Same thing here since `vlq_len` is not necessarily the length of the underlying memory